design.md

```diff
@@ -1,13 +1,25 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+Given the complexity of the problem, the user will be expected to be familiar with how picobot works. So, they will already need to know that picobot has 100 possible states, that it can only check if there are walls to the north / south / east / west. They syntax also looks kind of like python's, so people who are familiar with python might have a slight edge (it was not my original idea to steal Python's syntax, but I found it helped make my code a lot more compact and was one of the last changes I made to my desigh).
 
```

Comment:
When we looked at the API, it seemed like you could have more than 100 states, it was just in the DSL that CS5 used to write their codes that you were limited to 100 possible states.